src/AccessibilityInsights.CommonUxComponents/Dialogs/MessageDialog.xaml.cs

```diff
@@ -25,6 +25,7 @@ public MessageDialog()
         {
             var dlg = new MessageDialog() { Message = message };
             dlg.Owner = Application.Current.MainWindow;
+            dlg.Resources = Application.Current.MainWindow.Resources;
```

Comment:
Is there any implication from it? such as a certain resource is expected in App level resources?

Reply:
This is here to give the dialog access to the resources in Styles.xaml, specifically BtnBlue. In general, it gives the dialog access to whatever resources MainWindow has.

Reply:
If so, don't we need to consider splitting Styles.xaml into two parts(common and AI-Win specific)?

Reply:
Consider, yes. I would call that out of scope for this PR. There are more implications to deal with. Currently, SharedUx still owns theme, including styles, brushes, and font size. I'm not positive about separating those things out.